app/javascript/pulfalight/pulfalight_loader.es6

```diff
@@ -9,13 +9,23 @@ import AddToCartButton from '../components/AddToCartButton'
 import CartViewToggle from '../components/CartViewToggle'
 import TocBuilder from "../pulfalight/toc.es6"
 import LibCalHours from "../pulfalight/lib_cal_hours.es6"
+import QueryFiggy from "../pulfalight/query_figgy.es6"
 
 export default class PulfalightLoader {
   run() {
     this.setup_toc()
     this.setup_vue()
     this.setup_lib_cal_hours()
     this.setup_range_limit()
+    this.query_figgy()
+  }
+
+  query_figgy(){
+    const query = new QueryFiggy()
+    let doc_aspace_component_id = document.getElementById('document').getElementsByTagName('div')[0]
+    let component_id = doc_aspace_component_id.getAttribute('id').replace('doc_aspace_', '')
+    component_id = component_id.charAt(0).toUpperCase() + component_id.slice(1)
```

Comment:
Only the first char should be uppercase? Shouldn't any leading alphabetic characters be uppercased? I might move this line into `QueryFiggy` so it can be tested.

Reply:
Do the component_ids in figgy have more than one leading alphabetic character? If yes and if they are upper-cased then `any leading alphabetic character` should be uppercased. I can check on that as well.

Reply:
Yeah, for example `AC101`

Reply:
ok, I'll update. thanks